src/rpc/server.cpp

```diff
@@ -162,7 +162,7 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
         throw std::runtime_error(
             RPCHelpMan{"stop",
-                "\nStop Bitcoin server.",
+                "\nRequest a graceful shutdown of " PACKAGE_NAME ".",
```

Comment:
What's the point of having the package name here?

Reply:
Harmonization, consistency.